projects/plugins/jetpack/modules/module-extras.php

```diff
@@ -34,7 +34,6 @@
 	'external-media/external-media.php',
 	'plugin-search.php',
 	'scan/scan.php', // Shows Jetpack Scan alerts in the admin bar if threats found.
-	'simple-payments/simple-payments.php',
```

Comment:
Here I would recommend that you replace loading the file by loading the module file, `'simple-payments.php'`.